speedscale/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "display_name": "Speedscale",
+  "maintainer": "support@speedscale.com",
+  "manifest_version": "1.0.0",
+  "name": "speedscale",
+  "metric_prefix": "speedscale.",
+  "metric_to_check": "",
+  "creates_events": true,
+  "short_description": "Publish traffic replay results from Speedscale into Datadog.",
+  "guid": "f2e3ed64-a756-440f-9a1f-43089536aa08",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Speedscale",
+  "categories": [
+    "automation",
+    "containers",
+    "orchestration",
+    "testing"
+  ],
+  "type": "crawler",
+  "is_public": false,
```

Comment:
`is_public` is currently set to `false`, so this tile won't show up on Datadog docs. If you want this to appear in https://docs.datadoghq.com/integrations/, you should set this to `true` instead.

Reply:
Thank you for the catch, I just updated.